.travis.yml

```diff
@@ -18,6 +18,8 @@ install:
   - python setup.py -q develop
   - pip install -r requirements-test.txt
   - npm install -g jslint
+  # TODO: Remove when openwisp-users PR #123 is merged
+  - pip install --upgrade https://github.com/atb00ker/openwisp-users/tarball/swapper/
```

Comment:
Remove this change before merging, only for testing.

Reply:
changed to master